src/main/java/ch/wisv/areafiftylan/products/model/TicketBuyableDTO.java

```diff
@@ -0,0 +1,14 @@
+package ch.wisv.areafiftylan.products.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+public class TicketBuyableDTO {
+
+    @NotNull
+    private boolean buyable;
```

Comment:
A DTO with a single field means you don't really need a DTO, see Slack for details